src/app/components/ManualEntry.svelte

```diff
@@ -0,0 +1,133 @@
+<script lang="ts">
+  /**
+   * Handles the click when the delte button is pressed
+   * TODO
+   */
+  const handleDelete = (): void => {};
+</script>
+
+<style>
+  .language__sources-manual-entry {
+    border-radius: 20px;
+    padding: 5px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .language__sources-manual-entry-tablename input {
+    height: 30px;
+    width: 400px;
+    border-radius: 2px;
+    border: 1px solid var(--lite-white);
+    padding: 5px;
+    text-align: center;
+    font: 24px var(--main-font);
+    background-color: var(--lite-white);
+  }
+
+  th {
+    background-color: var(--lite-white);
+    padding: 20px 0px 20px 20px;
+    text-align: center;
+    vertical-align: middle;
+  }
+
+  table {
+    border-collapse: collapse;
+    width: 1400px;
+    margin-top: 30px;
+    margin-bottom: 50px;
+    box-shadow: 0px 25px 40px 0px rgba(0, 0, 0, 0.1);
+  }
+
+  tr {
+    border-bottom: 1pt solid var(--lite-white);
+  }
+
+  td {
+    padding: 10px 0px 10px 20px;
```

Comment:
Please follow the guidelines set here:  https://github.com/eddieantonio/predictive-text-studio/blob/production/docs/conventions.md#css-omit-the-unit-from-zero